REST_API_DOCS.md

```
@@ -952,50 +945,50 @@ Cmd + T (Ctrl + T) 누르면 탭 추가가 가능합니다. 참고하세요!
 {
   "status": "error",
   "error": {
     "code": "**UNAUTHORIZED**",
-    "message": "로그인 후 게시글 수정이 가능합니다.",
+    "message": "로그인 후 게시글 수정이 가능합니다."
   }
 }
 ```
 
-**Response (404 NOT FOUND)** 
+**Response (404 NOT FOUND)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)